examples/v2_with_iap/main.tf

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "cloud_run_v2" {
+  source  = "GoogleCloudPlatform/cloud-run/google//modules/v2"
+  version = "~> 0.16"
+
+  service_name = "ci-cloud-run-v2-iap"
+  project_id   = var.project_id
+  location     = "us-central1"
```

Comment:
@Deepraj1996 Are you aware which region has the most propagation delay? Can we use that region here?

Reply:
Thanks! Let's do this. Rest everything looks good to me.